components/navbar/chatlist.js

```diff
@@ -24,7 +24,11 @@ function ChatList({ currentUser, userID, friendsList, isDarkTheme }) {
     findMessagesPhotos()
     setTime()
     newList(found)
-}, [userFound], timer);
+    return () => {
+      userFound.current;
```

Comment:
I do not understand the use of a return statement in useEffect, nor the .current values being returned

Reply:
Talked to Isaac and he said this allows the refresh to only happen if the data is different than the current data the app already has